docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -9,19 +9,137 @@
 
 experimental[]
 
+
+Elastic Learned Sparse Encoder - or ELSER - is a representational model trained 
```

Comment:
```suggestion
Elastic Learned Sparse EncodeR - or ELSER - is a representational model trained 
```

For the R in ELSER. Is that correct @qherreros

Reply:
Yes, true!